prior_localization/run_scripts/format_outputs_stage_2.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import numpy as np
+import os
+import pandas as pd
+
+
+def custom_func(group):
```

Comment:
Please call the functions something more specific. Sorry to be petty, but it's the kind of thing that makes code easily look a little unprofessional